Dockerfiles/Dockerfile-tools

```
@@ -5,12 +5,17 @@ ARG VERSION
 # --------------------------------------------------------------------------------------------------
 # See ./builder for this image
 FROM cytopia/ansible-builder as builder
+ARG VERSION
 
 # Python packages (copied to final image)
 RUN set -eux \
-	&& pip3 install --no-cache-dir --no-compile \
-		dnspython \
-		mitogen \
+	&& if [ "${VERSION}" = "latest" ]; then \
+		pip3 install --no-cache-dir --no-compile dnspython mitogen; \
+	elif [ "${VERSION}" = "2.9" ]; then \
+		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
+	else \
+		pip3 install --no-cache-dir --no-compile dnspython mitogen; \
+	fi \
```

```suggestion
	&& if [ "${VERSION}" = "2.5" ]; then \
		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
	elif [ "${VERSION}" = "2.6" ]; then \
		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
	elif [ "${VERSION}" = "2.7" ]; then \
		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
	elif [ "${VERSION}" = "2.8" ]; then \
		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
	elif [ "${VERSION}" = "2.9" ]; then \
		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
	else \
		pip3 install --no-cache-dir --no-compile dnspython mitogen; \
	fi \
```

See here: https://github.com/mitogen-hq/mitogen/releases

* `Mitogen 0.2.10` Is for all Ansible versions < 2.10
* `Mitogen 0.3.0` Is for all Ansible versions >= 2.10